packages/data/addon/models/metadata/function-argument.ts

```diff
@@ -13,77 +13,77 @@ export const INTRINSIC_VALUE_EXPRESSION = 'self';
 export default class FunctionArgument extends EmberObject {
   /**
    * @static
-   * @property {String} identifierField
+   * @property {string} identifierField
    */
   static identifierField = 'id';
 
   /**
    * @property {Service} dimensionService
    */
   @service('bard-dimensions')
-  dimensionService;
+  dimensionService!: TODO;
 
   /**
-   * @property {String} id
+   * @property {string} id
    */
-  id;
+  id!: string;
 
   /**
-   * @property {String} name
+   * @property {string} name
    */
-  name;
+  name!: string;
 
   /**
-   * @property {String} description
+   * @property {string} description
    */
-  description;
+  description!: string;
 
   /**
-   * @property {String} source - name of the data source this argument is from.
+   * @property {string} source - name of the data source this argument is from.
    */
-  source;
+  source!: string;
 
   /**
    * @property {ValueType} valueType
    */
-  valueType;
+  valueType!: TODO;
 
   /**
-   * @property {String} type - either "ref" or "primitive"
+   * @property {string} type - either "ref" or "primitive"
    */
-  type;
+  type!: string;
 
   /**
-   * @property {String} expression - used if type is ref to get the valid values
+   * @property {string} expression - used if type is ref to get the valid values
    * Expected format is e.g. "dimension:dimensionOne" or "self" if the values come from an enum
    */
-  expression;
+  expression!: string;
 
   /**
    * @private
    * @property {String[]} _localValues
    * if metric function ids are not supplied by the metadata endpoint,
    * then enum values provided in the parameter will be placed here
    */
-  _localValues;
+  _localValues!: string[];
```

Comment:
```suggestion
  _localValues?: string[];
```
Will be undefined for function-arguments that get their values from a dimension